src/api/.rubocop.yml

```diff
@@ -55,3 +55,7 @@ Style/AndOr:
 
 Style/EmptyElse:
   EnforcedStyle: both
+
+Style/HashSyntax:
+  EnforcedStyle: ruby19
```

Comment:
I would prefer using `ruby19_no_mixed_keys`:

https://github.com/Ana06/rubocop/blob/patch-1/lib/rubocop/cop/style/hash_syntax.rb#L19

It is a link to a fork of the project in my account because the name of this option is wrong in the documentation of the Rubocop project (I've just open a PR).